lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -761,7 +759,8 @@ int amf_app_handle_pdu_session_accept(
   // Message construction for PDU Establishment Accept
   msg.security_protected.plain.amf.header.extended_protocol_discriminator =
       M5G_MOBILITY_MANAGEMENT_MESSAGES;
-  msg.security_protected.plain.amf.header.message_type = DLNASTRANSPORT;
+  msg.security_protected.plain.amf.header.message_type =
+      static_cast<M5GMessageType>(M5GMessageType::DLNASTRANSPORT);
```

Comment:
Casting "M5GMessageType" to "M5GMessageType"

Reply:
addressed review comment